pkg/matrix/matrix.go

```diff
@@ -191,6 +192,18 @@ func (me *Matrix) GetInstance(key string) *Instance {
 	return nil
 }
 
+func (me *Matrix) GetInstancesBySubkey(subKey string) []*Instance {
```

Comment:
would it be clearer if this method were named `GetInstancesBySuffix`?

Reply:
changed